client/blocks/reader-unsubscribed-feeds-search-list/reader-unsubscribed-feed-item.tsx

```diff
@@ -47,18 +46,15 @@ const ReaderUnsubscribedFeedItem = ( {
 	const translate = useTranslate();
 	const isEmailVerified = useSelector( isCurrentUserEmailVerified );
 	const dispatch = useDispatch();
-	const resendEmailVerification = useResendEmailVerification();
 	const filteredDisplayUrl = filterURLForDisplay( displayUrl );
 
 	const onSubscribeClickEvent = ! isEmailVerified
 		? () => {
 				dispatch(
-					errorNotice( translate( 'Your email has not been verified yet.' ), {
+					errorNotice( translate( 'Please verify your email before subscribing.' ), {
 						id: 'resend-verification-email',
```

Comment:
Should we change the ID here?

Reply:
No strong opinion. I have also thought about changing it but avoided it as the change seems unnecessary.